README.md

```
@@ -1,5 +1,11 @@
 # beagle
 
+> **Upgrading from beagle v1?** The monolithic `beagle` plugin was split into individual plugins in v2.0.0. If you still have `beagle@existential-birds` enabled in `~/.claude/settings.json`, it now loads as a no-op stub. You can remove it with:
+> ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)